src/xcode/ENA/ENA/Source/Scenes/Home/HomeTableViewController.swift

```diff
@@ -56,6 +56,14 @@ class HomeTableViewController: UITableViewController, NavigationBarOpacityDelega
 				self?.viewModel.isUpdating = false
 			}
 			.store(in: &subscriptions)
+		
+		viewModel.$healthCertifiedPersons
+			.receive(on: DispatchQueue.OCombine(.main))
+			.sink { [weak self] _ in
+				self?.tableView.reloadSections([HomeTableViewModel.Section.healthCertificate.rawValue], with: .none)
+				self?.viewModel.isUpdating = false
```

Comment:
Does this have any effect?  Where is it set to true for the case of healthCertifiedPersons?